api/agentpb/agent.proto

```diff
@@ -67,6 +67,8 @@ message SetStateRequest {
   message BuiltinAgent {
     inventory.AgentType type = 1;
     string dsn = 2;
+    // Limit query length in QAN (default: server-defined; -1: no limit).
+    int32 query_length = 8;
```

Comment:
`max_query_lenght`?

Reply:
Renamed to maxQueryLength on all places (will do same for postgres PR), because its better naming.